tests/tests_custom_certificate.yml

```
@@ -0,0 +1,49 @@
+---
+
+- name: Test PostgreSQL server with ssl support using certificate role
+  hosts: all
+  tasks:
+    - name: Test PostgreSQL server user certificate in custom path
+      vars:
+        postgresql_password: redhat
+      block:
+        - name: Generate certificate using certificate role
+          include_role:
+            name: fedora.linux_system_roles.certificate
+          vars:
+            certificate_requests:
+              - name: /etc/pki/tls/certs/mycert
```

```suggestion
              - name: /etc/pki/tls/certs/postgresql_test
```
So that if something goes wrong with the test, and doesn't clean up, we know which test this is